.github/workflows/test.yml

```diff
@@ -2,6 +2,9 @@ name: Tests
 on: [push, pull_request]
 jobs:
   test:
+    strategy:
+      matrix:
+        java_version: [17, 19, 21, 24]
```

Comment:
Changes to `test.yml` aren't necessary for the acceptor/selector options, right?